package.json

```diff
@@ -71,4 +71,4 @@
     "node": ">= 0.10"
   },
   "license": "MIT"
-}
\ No newline at end of file
+}
```

Comment:
Can you revert your changes to package.json, i guess your code editor added a new line here !

Reply:
i'll make the changes and yes my contribution is related to hacktoberfest